internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -311,3 +333,43 @@ func setCPUQuota(cpuMountPoint, parentDir string, c *oci.Container, enable bool)
 
 	return nil
 }
+
+// RestoreIrqBalanceConfig restores irqbalance service with original banned cpu mask settings
+func RestoreIrqBalanceConfig(irqBalanceConfigFile, irqBannedCPUConfigFile string) error {
```

Comment:
@cynepco3hahue Thanks for your comments! 
If this happens, can we just rely on `irqSmpAffinityProcFile "/proc/irq/default_smp_affinity"` and restore the config only if this affinity file contains zero value (i.e. system reboot scenario) ?

Reply:
@cynepco3hahue @haircommander have added the logic to restore the config only if default smp affinity proc file has every cpu mask set, this could be a simple approach if none of other applications in the system modified that file beforehand.